LoopKitTests/InsulinMathTests.swift

```diff
@@ -277,8 +277,8 @@ class InsulinMathTests: XCTestCase {
     }
     
     func testInsulinOnBoardLimitsForExponentialModel() {
-        let insulinModel = ExponentialInsulinModel(actionDuration: TimeInterval(minutes: 360), peakActivityTime: TimeInterval(minutes: 75))
-        let childModel = ExponentialInsulinModel(actionDuration: TimeInterval(minutes: 360), peakActivityTime: TimeInterval(minutes: 65))
+        let insulinModel = ExponentialInsulinModel(actionDuration: TimeInterval(minutes: 360), peakActivityTime: TimeInterval(minutes: 75), delay: TimeInterval(minutes: 0))
+        let childModel = ExponentialInsulinModel(actionDuration: TimeInterval(minutes: 360), peakActivityTime: TimeInterval(minutes: 65), delay: TimeInterval(minutes: 0))
```

Comment:
I would prefer to see some unit tests that include a non-zero `delay` also.

Reply:
The tests that do not specify delay use the default of 10 minutes.

Reply:
But, since the logic changed, why are there no tests to check the new math?

Reply:
The ability to do a delay is not new, what's changed is where the delay is specified, and where it's applied. We do have tests of the models using both 0m and 10m delays, here, though, as we did before.

Reply:
Ah ok well that explains it.  I didn't know it was covered. Thanks for clarifying!